odoo/addons/base/views/ir_filters_views.xml

```diff
@@ -5,24 +5,41 @@
             <field name="model">ir.filters</field>
             <field name="arch" type="xml">
                 <form string="Filters">
-                  <sheet>
-                    <group col="4">
-                        <field name="name"/>
-                        <field name="user_id" string="Available for User"/>
-                        <field name="model_id"/>
-                        <field name="is_default"/>
-                        <field name="action_id"/>
-                        <field name="active" widget="boolean_toggle"/>
-                    </group>
-                    <group>
-                        <field name="domain" widget="domain" options="{'model': 'model_id'}"/>
-                        <field name="context"/>
-                        <field name="sort"/>
-                    </group>
-                   </sheet>
+                    <field name="model_id" invisible="1"/> <!-- required for the domain widget model option when the user is not in debug -->
+                    <sheet>
+                        <div class="ribbon ribbon-top-right" invisible="active">
+                            <span class="text-bg-primary">Archived</span>
+                        </div>
+                        <group>
+                            <group>
+                                <field name="name"/>
+                                <field name="user_id" string="Owner" placeholder="All users"/>
+                                <field name="is_default" widget="boolean_toggle"/>
+                                <field name="model_id" groups="base.group_no_one"/>
```

Comment:
do the attributes group should be put only in the extension? This view is only accessible in debug mode I guess